### ISSUE TITLE
Duplicated output

### ISSUE THREAD

**Issue by Helveg** [essenciary/Genie.jl#117]
As seen in the 2 other issues I've opened my Windows 10 machine duplicates some command line output. It seems non-problematic although Genie has not behaved 100%. Maybe it's ever of use in solving another issue:

```
2019-02-19 09:03:38:DEBUG:Main: Web Server starting at 127.0.0.1:8000
2019-02-19 09:03:38:DEBUG:Main: Web Server running at 127.0.0.1:8000
2019-02-19 09:03:38:DEBUG:Main: Web Server running at 127.0.0.1:8000
```

```
2019-02-19 09:01:25:INFO:Main: Done! New app created at C:\Users\Robin\GIT\plot-repo\some-thing
2019-02-19 09:01:25:INFO:Main: Done! New app created at C:\Users\Robin\GIT\plot-repo\some-thing
2019-02-19 09:01:25:DEBUG:Main: Changing active directory to some-thing
2019-02-19 09:01:25:DEBUG:Main: Changing active directory to some-thing
2019-02-19 09:01:25:DEBUG:Main: Installing app dependencies
2019-02-19 09:01:25:DEBUG:Main: Installing app dependencies
  Updating registry at `C:\Users\Robin\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
2019-02-19 09:01:30:INFO:Main: Starting your brand new Genie app - hang tight!
2019-02-19 09:01:30:INFO:Main: Starting your brand new Genie app - hang tight!
```

versioninfo()

```
Julia Version 1.0.0
Commit 5d4eaca0c9 (2018-08-08 20:58 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: AMD A9-9425 RADEON R5, 5 COMPUTE CORES 2C+3G
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, btver1)
```

**Comment by essenciary** [comment]
@Helveg Thanks - do you always get the duplicated output? 
Can you please include the output from `pkg> st`

**Comment by Helveg** [comment]
It varies between 1 to 3 duplicates, I haven't really been able to see a pattern but it seems pretty stable if I repeat the same command multiple times and it varies if I for example close and open Atom, or command line prompts, ...l

Here is st:

```
    Status `C:\Users\Robin\.julia\environments\v1.0\Project.toml`
  [c52e3926] Atom v0.7.14
  [31c24e10] Distributions v0.16.4
  [c43c736e] Genie v0.1.0 #master (https://github.com/essenciary/Genie.jl)
  [682c06a0] JSON v0.20.0
  [e5e0dc1b] Juno v0.5.4
  [91a5bcdd] Plots v0.23.0
  [3cdcf5f2] RecipesBase v0.6.0
  [295af30f] Revise v1.0.2
```

Revise wasn't installed as a dependency when generating a new Genie app btw, I had to do that manually.

**Comment by essenciary** [comment]
I can't reproduce - it would be helpful if you could provide more details so I can do the same. It looks like the Logger is registered multiple times, but I'm not sure how. On my Win 10 it works as expected: 
<img width="1163" alt="image" src="https://user-images.githubusercontent.com/11292378/53128629-f931ab00-356d-11e9-845b-5eb72e50e339.png">

Anyway, as mentioned, please update to latest Genie. That definitely lists Revise as a dependency for apps -- and it might address the logging issue.

**Comment by essenciary** [comment]
@Helveg I accidentally managed to get the duplicate log output. Not sure why it happened but I think I figured out what's causing it. So I improved the logging module and it should not duplicate the output anymore. If you have a chance to test it, please let me know if it's fixed or not (requires latest genie, v0.8.6). Thanks
